shotover-proxy/src/transforms/kafka/sink_single.rs

```diff
@@ -111,11 +113,28 @@ impl Transform for KafkaSinkSingle {
         let responses = responses?;
 
         // TODO: since kafka will never send requests out of order I wonder if it would be faster to use an mpsc instead of a oneshot or maybe just directly run the sending/receiving here?
-        if let Some(read_timeout) = self.read_timeout {
+        let mut responses = if let Some(read_timeout) = self.read_timeout {
             timeout(read_timeout, read_responses(responses)).await?
         } else {
             read_responses(responses).await
+        }?;
+
+        for response in &mut responses {
```

Comment:
Can we call this `message`, I thought it was a ResponseBody type instead.

Reply:
it is a response from the list of responses.
Naming `response` -> `message` seems strange, should we also do `responses` -> `messages` as well?

Reply:
Just used to seeing `message.frame()`, not too fussed though.